index.html

```diff
@@ -231,6 +231,35 @@ <h1 style="margin: 0.2em 0;font-size: 3em;font-weight: 300;">Roadmap<a class=" s
                     and <a href="https://github.com/scalameta/scalameta/issues/359">#359</a>
                     on our issue tracker if that's important for you.
                 </p>
+                <div style="margin: 0px;color: #333;text-align: center;padding: 2.5em 2em 0;border-bottom: 1px solid #eee;display: block;" id="FAQ" class=" scalatex-site-Styles-hoverContainer scalatex-site-Styles-headerTag">
+                    <h1 style="margin: 0.2em 0;font-size: 3em;font-weight: 300;">FAQ
+                        <a class=" scalatex-site-Styles-headerLink" href="#FAQ" style="position: absolute;right: 0px;"><i class="fa fa-link"></i></a>
+                    </h1>
+                    <br />
+                </div>
+                <p>
+                    <b>Does Meta support X Dialect</b>
```

Comment:
We could make it a table like this

<img width="816" alt="screen shot 2017-08-23 at 08 36 53" src="https://user-images.githubusercontent.com/1408093/29601943-4241a67c-87de-11e7-91cd-da7944e6a452.png">